future-apps/payroll/app/src/screens/components/AvailableSalary.js

```diff
@@ -3,6 +3,7 @@ import styled from 'styled-components'
 import AvailableSalaryTable from './AvailableSalaryTable'
 import { formatCurrency, SECONDS_IN_A_YEAR } from '../../utils/formatting'
 import { differenceInSeconds } from 'date-fns'
+import { summation } from '../../utils/calculations'
```

Comment:
This could be changed to `utils/calculations` based on https://parceljs.org/module_resolution.html

Reply:
I have replaced all relatives paths to tilde parcel operator in all file in this PR